kube/services/fence/fence-deploy.yaml

```diff
@@ -134,6 +157,8 @@ spec:
 # DEPRECATED! Remove when all commons are no longer using local_settings.py
 #             for fence.
 # -----------------------------------------------------------------------------
+          - name: logs-folder
+            mountPath: "/var/log/nginx/"
```

Comment:
Are these logs being rotated?

Reply:
Nginx does not provide a mechanism to rotate logs (according to some quick googling).
Do you have any ideas?
How about a cronjob to run some logrotate?